wpcom-helper.php

```diff
@@ -24,14 +29,27 @@ function jetpack_amp_disable_the_content_filters( $post_id ) {
 
 add_action( 'amp_post_template_head', 'jetpack_amp_add_og_tags' );
 
-function jetpack_amp_add_og_tags( $amp_template ) {
+/**
+ * Add Open Graph tags.
+ *
+ * @since 0.3
+ */
+function jetpack_amp_add_og_tags() {
 	if ( function_exists( 'jetpack_og_tags' ) ) {
 		jetpack_og_tags();
 	}
 }
 
 add_filter( 'amp_post_template_metadata', 'jetpack_amp_post_template_metadata', 10, 2 );
 
+/**
+ * Add publisher and image metadata.
+ *
+ * @param array   $metadata Metadata array.
+ * @param WP_Post $post     Post.
+ *
+ * @return array Modified metadata array.
+ */
 function jetpack_amp_post_template_metadata( $metadata, $post ) {
 	if ( isset( $metadata['publisher'] ) && ! isset( $metadata['publisher']['logo'] ) ) {
 		$metadata = wpcom_amp_add_blavatar_to_metadata( $metadata, $post );
```

Comment:
It'd probably be good to remove the second argument `$post` here, as the function function [now only accepts](https://github.com/Automattic/amp-wp/pull/1058/files#diff-e3795a514c859a54511c3580d489cab3R73) one argument.

Reply:
Good catch. Fixed in f683d9f.